build-tools/automation/azure-pipelines.yml

```diff
@@ -0,0 +1,223 @@
+      Configuration: Release
+      JavaSdkDirectory: /Library/Java/JavaVirtualMachines/jdk1.8.0_144.jdk/Contents/Home/
+      JAVA_HOME: $(JavaSdkDirectory)
+    steps:
+    - task: DownloadPipelineArtifact@1
+      inputs:
+        artifactName: $(BundleArtifactName)
+        downloadPath: $(System.DefaultWorkingDirectory)
+
+    - script: make prepare-external-git-dependencies CONFIGURATION=$(Configuration)
+      displayName: make prepare-commercial
+      condition: eq(variables['XA.Commercial.Build'], 'true')
+      env:
+        GH_AUTH_SECRET: $(Github.Token)
+
+    - task: provisionator@2
+      condition: and(succeeded(), eq(variables['XA.Commercial.Build'], 'true'))
+      inputs:
+        provisionator_uri: $(provisionator-uri)
+        github_token: $(GitHub.Token)
+        provisioning_script: $(System.DefaultWorkingDirectory)/external/monodroid/build-tools/provisionator/profile.csx
+        provisioning_extra_args: -vv
+
+    - script: make prepare-deps V=1 CONFIGURATION=$(Configuration) MSBUILD_ARGS="$(AutoProvisionArgs)"
+      displayName: make prepare-deps
+
+    - script: make prepare jenkins V=1 CONFIGURATION=$(Configuration) MSBUILD_ARGS="$(AutoProvisionArgs) /p:BundleRootPath=$(System.DefaultWorkingDirectory)"
+      displayName: make prepare jenkins
+
+    - script: make create-installers V=1 CONFIGURATION=$(Configuration)
+      displayName: make create-installers
+
+    - script: |
+        mkdir -p $(System.DefaultWorkingDirectory)/unsigned-installers
+        cp bin/Build$(Configuration)/*.vsix $(System.DefaultWorkingDirectory)/unsigned-installers
+        cp bin/Build$(Configuration)/*.pkg $(System.DefaultWorkingDirectory)/unsigned-installers
+      displayName: copy build artifacts
+
+    - task: PublishPipelineArtifact@0
+      displayName: upload unsigned installers
+      inputs:
+        artifactName: unsigned
+        targetPath: $(System.DefaultWorkingDirectory)/unsigned-installers
+
+    - task: MSBuild@1
+      displayName: package build results
+      inputs:
+        solution: build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks.csproj
+        configuration: $(Configuration)
+        msbuildArguments: /t:ZipBuildStatus /p:ResultPackageOutputPath=$(Build.ArtifactStagingDirectory)
+      condition: always()
+
+    - task: PublishPipelineArtifact@0
+      displayName: upload artifacts
+      inputs:
+        artifactName: mac-build-results
+        targetPath: $(Build.ArtifactStagingDirectory)
+      condition: always()
+
+- stage: win_build_test
+  displayName: Win
+  dependsOn: prepare
+  variables:
+    Configuration: Release
+  jobs:
+  - job: build_test
+    displayName: Build and Test
+    pool: VSEng-MicroBuildVS2017
+    timeoutInMinutes: 360
+    cancelTimeoutInMinutes: 5
+    variables:
+      _AndroidDesignTimeResDirIntermediate: .
+      CleanAllFiles: False
+      CleanOnNuGetPackagesChange: False
+      NuGet.Show.Stack: true
+      NUnit.Extra: --trace=Verbose
```

Comment:
All of these `variables` look they were added for debugging purposes. Let's try removing them all?

If I need these again, I would probably set them at queue-time. I either didn't know how to do that yet, or it was a recent feature.

Reply:
I've removed all of the extra vars on this job.